app/controllers/spree/admin/products_controller.rb

```diff
@@ -82,14 +82,9 @@ def bulk_update
 
       def clone
         @new = @product.duplicate
+        @new.save
 
-        flash[:success] = if @new.save
-                            Spree.t('notice_messages.product_cloned')
```

Comment:
Removing this to match with old UI experience.

Reply:
Good idea to remove the unused code 👍 
Of course, we will try to implement these messages in the new UI, but the method will depend on https://github.com/openfoodfoundation/openfoodnetwork/issues/11517 which is in progress.